addons/stock/models/stock_picking.py

```diff
@@ -160,20 +160,18 @@ def _compute_picking_count(self):
             for record in self:
                 record[field_name] = count.get(record.id, 0)
 
-    def name_get(self):
+    def _compute_display_name(self):
         """ Display 'Warehouse_name: PickingType_name' """
-        res = []
         for picking_type in self:
             if picking_type.warehouse_id:
-                name = picking_type.warehouse_id.name + ': ' + picking_type.name
+                name = f"{picking_type.warehouse_id.name}: {picking_type.name}"
```

Comment:
Can assign the `picking_type.display_name` directly, the `name` variable is useless here, and so, one line less diff ! I know you like that :smirk: